inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -54,6 +55,8 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     OV_ITT_SCOPED_TASK(itt::domains::IETransform, "ngraph::pass::CommonOptimizations");
 
     ngraph::pass::Manager manager;
+    // Propagate local PassConfig to internal pass::Manager
+    manager.set_pass_config(get_pass_config());
```

Comment:
@GlebKazantaev @lazarevevgeny , could you please review this change. I move up this line from after all register_pass<>(). The reason is: if set local PassConfig to manager in the end, which means **reset** passconfig, some configuration introduced in register_pass<>() is lost. e.g:
`manager.register_pass<ngraph::pass::ConvertInterpolate1ToInterpolate4, false>();` this pass is disabled by default and noted in manager.passconfig. It's lost after reset with config of passBase. I change to reset in the beginning. The new config is appended or config is modified in each register_pass<>(), not reset instead. The same changes for opset 3-2/ 2-1/ 1-legecy transformations.

Reply:
@chenhu-wang yes, you are right set_pass_config will override all disabled passes so your change is valid. But anyway I will create a bug because I expected that setting new config doesn't depends on its order.

Reply:
@GlebKazantaev ,I have reverted this workaround and will rebase on your the separate fix:https://github.com/openvinotoolkit/openvino/pull/2940. Thank you!

Reply:
@GlebKazantaev , I have rebased on the your fix. All works fine. Thanks you! Do you have any more comments?